### DISCUSSION TITLE
package.json 이 없다고 나옵니다.

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#265] [5. Q&A]
왜 그런지 모르겠는데. 
월드를 생성하면 package.json 이 없다고 나옵니다. 물론 zepeto 템플릿 프로젝트로 했구요. 
관련 폴더를 보면 이전에 성공했을시 있었던 파일들이 없습니다. (package.json, tsconfig.json)
관련 파일을 복사하면 되는지, 아니면 생성하는 방법이 있는지 모르겠네요.. 여러번 만들어 봐도 마찬가지 입니다. 

허브 3.0.0 을 사용하는데 허브가 문제일리는 없을것 같은데...

**Comment by DoLongJu** [comment]
자세한 내용은 모르겠지만 유니티 허브 2.4.5를 이용해 보시겠어요? 저는 허브 3.0.0으로 설치했다가 다시 내렸습니다 ㅎㅎ..

**Comment by moondory77-naverz** [comment]
안녕하세요. 
ZEPETO 템플릿 프로젝트를 시작할 때, 노드환경구성에 실패한 것으로 보이는데요..

문제상황에 대해, 조금 더 범위를 좁힐 수 있는 단서를 주시면 감사하겠습니다.

1. ZEPETO 템플릿 프로젝트를 생성하는 환경
2. (이전에 프로젝트 생성이 가능했다면) 환경구성에 차이가 있는지
3. 템플릿 프로젝트를 시작할 때 발생하는 에러메시지 등..
(단서가 될만한 건 무엇이든 좋습니다.)

감사합니다.
